vcmi-updates.json

```diff
@@ -21,6 +21,9 @@
 		"VCMI 1.1.1",
 		"VCMI 1.2.0.c125e040c305c88f66e7ba8a458ee22e4d455f30",
 		"VCMI 1.2.0",
-		"VCMI 1.2.1.25d5a1555cf51663bb8d8409dbc269dfaf511da8"
+		"VCMI 1.2.1.25d5a1555cf51663bb8d8409dbc269dfaf511da8",
+		"VCMI 1.2.1",
+		"VCMI 1.3.0.b9729241f1018b4a07f880b319da32f94bf6409b",
+		"VCMI 1.3.0"
```

Comment:
it's actually not necessary, because 1.3.0 is latest and update from it is not required. However, code has check for currentVersion == newVersion so fine to have